crates/extensions/tedge_health_ext/src/lib.rs

```diff
@@ -40,6 +41,7 @@ impl HealthMonitorBuilder {
         // TODO: pass it less annoying way
         mqtt_schema: &MqttSchema,
         service_type: String,
+        topic_root: String,
```

Comment:
The same argument is given twice: first as an `MqttSchema`, second as `String`. Keep only the first one.

Reply:
fixed